### PR TITLE
Issue39 gs6 add params to ingestors

### PR DESCRIPTION
**Pull request by aanersc**

### CODE REVIEW

**Comment by aanersc** [comment]
@akorosov 
By adding a line into the loop, we can create the relation between the parameter and the dataset.
```
ds.parameters.add(pp)
```
based on it, we can call the desired filtering as mentioned in the beginning of the issue:
image


Please find the new code in the a7f9ad0

File: geospaas/nansat_ingestor/managers.py [hunk 1]
```diff
@@ -127,12 +131,35 @@ def get_or_create(self, uri, n_points=10, uri_filter_args=None, *args, **kwargs)
...
+        # After fixing the issue in Nansat, the codes can be replaced by the one commented above.
+        # create parameter
+        ###all_band_meta = n.bands()
+        ###for band_id in range(1, len(all_band_meta)+1):
+        ###    band_meta = all_band_meta[band_id]
+        ###    if 'standard_name' in band_meta.keys():
+        ###        pp = Parameter.objects.get(standard_name=band_meta['standard_name'])
+        ###        dsp, dsp_created = DatasetParameter.objects.get_or_create(dataset=ds, parameter=pp)
+
+
```
Comment (akorosov) [inline geospaas/nansat_ingestor/managers.py:160]:
```suggestion
```

File: geospaas/nansat_ingestor/managers.py [hunk 2]
```diff
@@ -114,6 +111,13 @@ def get_or_create(self, uri, n_points=10, uri_filter_args=None, *args, **kwargs)
         geolocation = GeographicLocation.objects.get_or_create(
                       geometry=WKTReader().read(n.get_border_wkt(nPoints=n_points)))[0]
 
+        # create parameter
+        from geospaas.vocabularies.models import Parameter
```
Comment (akorosov) [inline geospaas/nansat_ingestor/managers.py:115]: Move import to top

File: geospaas/nansat_ingestor/managers.py [hunk 3]
```diff
@@ -114,6 +111,13 @@ def get_or_create(self, uri, n_points=10, uri_filter_args=None, *args, **kwargs)
...
+        nansat_bands = n.bands()
+        for band_number in range(1, len(nansat_bands)+1):
+            band_dict = nansat_bands[band_number]
+            if 'standard_name' in band_dict.keys():
+                parameter = Parameter.objects.get_or_create(nansat_bands[band_number])[0]
```
Comment (akorosov) [inline geospaas/nansat_ingestor/managers.py:120]: Remove the repetition

File: geospaas/nansat_ingestor/managers.py (hunk 2 above)
Comment (aanersc) [reply geospaas/nansat_ingestor/managers.py:115]: done

**Comment by aanersc** [comment]
@akorosov 
ready to be merged by f2b20d8. Mentioned files are deleted by git rm command.

**Pull request merged**